### ISSUE TITLE
Implement execute_command_line("...")

### ISSUE THREAD

**Issue by certik** [lfortran/lfortran#1053]
Put it in src/runtime/impure/, call into C and register it like other intrinsics.

**Comment by rootxrishabh** [comment]
Hey @certik, I am really new to open source, so just to be sure I have to put 'execute_command_line("...")' into [this](https://github.com/rootxrishabh/lpython/blob/main/src/runtime/impure/lfortran_intrinsic_sin_c.c). Right?

**Comment by HarshvMahawar** [comment]
if no one is working on this issue can I try it?  Im new here so can you elaborate what exactly has to be done

**Comment by HarshitaKalani** [comment]
This can be closed as it is implemented already.

**Comment by gxyd** [comment]
Closing this issue as it has been implemented already.

**Comment by gxyd** [comment]
Yes, I'm taking a look that. I checked that the implementation in runtime isn't there. Should I still keep the issue open then?
